content/docs/user-guide/editor/customizing.md

```diff
@@ -85,58 +76,51 @@ You can change the general O3DE Editor settings and file settings.
 | Show Time in Console |  Displays the time in the console window.  |
 | Toolbar Icon Size |  Adjusts the toolbar icon size.  Default: `16` (32 pixels).  |
 | Stylus Mode |  Enables stylus mode for tablets and other pointing devices.  |
-|  **Enable UI 2.0 (EXPERIMENTAL)**  |  Enables the updated user interface (UI) in O3DE Editor.  |
-|  **Enable Scene Inspector (EXPERIMENTAL)**  | Enables the option to inspect scenes in files such as .fbx files. |
-|  **Restore Viewport Camera on Game Mode Exit**  |  Returns the camera to the original transform when you exit gameplay mode.  |
-|  **Enable Legacy UI (DEPRECATED)**  |  Enable the legacy user interface (UI) in O3DE Editor.  |
-|  **Enable New Viewport Interaction Model (EXPERIMENTAL)**  |  Replaces the old interaction model in the viewport. For more information, see [Working with the Viewport Interaction Model](/docs/user-guide/editor/viewport/).  |
-
+| Enable UI 2.0 (EXPERIMENTAL) |  Enables the updated user interface (UI) in O3DE Editor.  |
+| Enable Scene Inspector (EXPERIMENTAL) | Enables the option to inspect scenes in files such as .fbx files. |
+| Restore Viewport Camera on Game Mode Exit |  Returns the camera to the original transform when you exit gameplay mode.  |
+| Enable Legacy UI (DEPRECATED) |  Enable the legacy user interface (UI) in O3DE Editor.  |
+| Enable New Viewport Interaction Model (EXPERIMENTAL) |  Replaces the old interaction model in the viewport. For more information, see [Working with the Viewport Interaction Model](/docs/user-guide/editor/viewport/).  |
```

Comment:
update: I think these should be removed to (Legacy UI is no longer supported and the new Viewport Interaction Model _is_ the viewport, so enabling/disabling no longer makes sense).

Reply:
Done.

Reply:
Perfect thanks!